src/measure.js

```diff
@@ -0,0 +1,27 @@
+const { get, is, em, num, breaks, dec, media, merge } = require('./util')
+const { measures } = require('./constants')
+const width = require('./width')
+
+module.exports = props => {
+  const n = props.measure
+
+  if(!is(n)) return null
+
+  const scale = get(props, 'theme.measures', measures)
+
+  if (!Array.isArray(n)) {
+    return {
+      width: mx(scale)(n)
```

Comment:
I feel like this should use `max-width` and the utility should be called `maxWidth` – sticking to CSS naming conventions like the rest of (most of??) the library

Reply:
Yeah, `max-width` makes sense to me. I've updated my pr to use the css naming.